tests/lms/views/decorators/test_h_api.py

```diff
@@ -295,6 +322,10 @@ def wrapped():
 @pytest.fixture
 def hapi_svc(patch, pyramid_config):
     hapi_svc = mock.create_autospec(HypothesisAPIService, spec_set=True, instance=True)
+    hapi_svc.patch.return_value = mock.create_autospec(
+        Response, instance=True, status_code=200, reason="OK", text=""
+    )
+    hapi_svc.patch.return_value.json.return_value = {"id": "TEST_PUBID"}
```

Comment:
This line isn't needed (the lms app doesn't use pubids anymore). There's a pr to remove the  preexisting pubid return value below too